opendevin/lib/actions/util.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# This is the path where the workspace is mounted in the container
+# The LLM sometimes returns paths with this prefix, so we need to remove it
+PATH_PREFIX = "/workspace/"
+
+def resolve_path(base_path, file_path):
+    if file_path.startswith(PATH_PREFIX):
```

Comment:
I wonder if the LLM can also return "/file/path". The word "sometimes" is a tad worrying, hah. But maybe further tweaks are good for a first issue.

Reply:
Yeah we can update down the line if needed. I'm going to merge as it stands now.